src/landscape/C4LandscapeRender.cpp

```diff
@@ -71,6 +71,9 @@ bool C4LandscapeRenderGL::Init(int32_t iWidth, int32_t iHeight, C4TextureMap *pT
 	this->iHeight = iHeight;
 	this->pTexs = pTexs;
 
+	// Start timer
+	this->TimerStart = std::chrono::system_clock::now();
```

Comment:
`std::chrono::system_clock` isn't guaranteed to be monotonic; use `std::chrono::steady_clock` instead.